<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="SomaSekhar-PC" name="com.fileupload.ReportUpload" tests="1" failures="0" timestamp="17 Dec 2018 09:58:15 GMT" time="0.002" errors="1">
  <testcase name="uploadFile" time="0.002" classname="com.fileupload.ReportUpload">
    <error type="java.lang.Error" message="Unresolved compilation problems: 
	CertificateException cannot be resolved to a type
	CertificateException cannot be resolved to a type
	Unhandled exception type NoSuchAlgorithmException
	Unhandled exception type KeyManagementException
">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
CertificateException cannot be resolved to a type
CertificateException cannot be resolved to a type
Unhandled exception type NoSuchAlgorithmException
Unhandled exception type KeyManagementException

at com.fileupload.MultipartUtility.<init>(MultipartUtility.java:55)
at com.fileupload.FileUploaderClient.uploadFile(FileUploaderClient.java:16)
at com.fileupload.ReportUpload.uploadFile(ReportUpload.java:58)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:425)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- uploadFile -->
</testsuite> <!-- com.fileupload.ReportUpload -->
